BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -14,8 +14,27 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define RSA_KEY_VB_SIGNATURE     SIGNATURE_32 ('$', '_', 'V', 'B')
 #define RSA_KEY_IPP_SIGNATURE    SIGNATURE_32 ('$', 'I', 'P', 'P')
+#define PUBKEY_IDENTIFIER        SIGNATURE_32 ('P', 'U', 'B', 'K')
+#define SIGNATURE_IDENTIFIER     SIGNATURE_32 ('$', 'I', 'P', 'P')
+
 
 #define SIG_TYPE_RSA2048SHA256   0
+#define SIG_TYPE_RSA3072SHA384   1
+
+#define SIGNING_TYPE_RSA_PKCS_1_5     1
+#define SIGNING_TYPE_RSA_PSS          2
+#define SIGNING_TYPE_ECC              3
```

Comment:
Let's skip adding these SIGNING_TYPE_ECC and DSA for now. In future, when we support ECC keys, the signing method we may support is ECDSA and then we can add the signing method.

Reply:
done